tests/test_non_ascii_search_string/code/dot_c.c

```diff
@@ -0,0 +1 @@
+// Just a .c file to test find-by-extension
```

Comment:
Of course. I'll work on it. And thanks for the information. So let's close this PR. We meet in the next one. : )